tendermint/Cargo.toml

```diff
@@ -47,9 +47,9 @@ serde_json = { version = "1" }
 serde_bytes = "0.11"
 serde_repr = "0.1"
 sha2 = { version = "0.8", default-features = false }
```

Comment:
It'd be good to bump this (and the other `digest`-based) dependencies as well. Could be done in a separate PR.